corehq/apps/app_manager/app_translations/app_translations.py

```diff
@@ -637,14 +637,29 @@ def is_markdown_vetoed(text_node_):
             text_node = itext.find("./{f}translation[@lang='%s']/{f}text[@id='%s']" % (lang, label_id))
             vetoes[label_id] = vetoes[label_id] or is_markdown_vetoed(text_node)
             markdowns[label_id] = markdowns[label_id] or had_markdown(text_node)
-
+    # skip labels that have no translation provided
+    skip_label = set()
+    for row in rows:
+        for lang in app.langs:
+            if row.get(_get_col_key('default', lang)):
+                break
+        else:  # https://stackoverflow.com/a/654002
```

Comment:
Linking to SO to explain a construct is a code smell. Why not just rewrite using an inner function?

Reply:
Cool, updated.